tests/arch/arm/arm_irq_vector_table/src/arm_irq_vector_table.c

```diff
@@ -16,8 +16,8 @@
  */
 #define _ISR_OFFSET 0
 
-#if defined(CONFIG_SOC_SERIES_NRF52X)
-/* The customized solution for nRF52X-based platforms
+#if defined(CONFIG_SOC_SERIES_NRF51X) || defined(CONFIG_SOC_SERIES_NRF52X)
+/* The customized solution for nRF5X-based platforms
```

Comment:
Same correction needed in [line 26](https://github.com/zephyrproject-rtos/zephyr/pull/19856/files#diff-c64ef6ac0266b8f4ff2c414cd2733cdeR26) (can't add a comment there).

Reply:
Thank, will add